### DISCUSSION TITLE
wrong location when verifying student status

### DISCUSSION THREAD

**Discussion by ghost** [community/community#18929] [GitHub Education]
Hello,

When I try to apply for student pack, it says "The school you selected does not appear to have a campus location in your country". I think it is because I use VPN to access GitHub quickly. As a result, it seems that I am in Singapore. However, I am living in China right now. After I terminated my VPN, it still says my location is wrong. Could you please fix my application? Thank you.

**Comment by songwookim** [comment]
did you get solving this? go to "Settings-email" and add your student email.  you'll get an email from github

**Comment by melvincwng** [comment]
Hi @sam2259, you can raise a ticket with GitHub Support right here - https://support.github.com/contact/education

They usually respond within 1-2 days depending on the number of support tickets.

Hope this clarifies.
